plugins/CoreVisualizations/CoreVisualizations.php

```diff
@@ -37,6 +39,21 @@ public function deleteUser($userLogin)
         ViewDataTableManager::clearUserViewDataTableParameters($userLogin);
     }
 
+    public function addViewDataTable(&$viewDataTable)
+    {
+        if (Common::getRequestVar('pivotBy', '')) {
+            $tableToRemove = 'Visualizations\HtmlTable';
+        } else {
+            $tableToRemove = 'HtmlTable\PivotBy';
```

Comment:
Needed to make Pivot work again as it only worked because of a random filter order before. Pivot has now it's own visualization and will be applied automatically if needed